book.toml

```diff
@@ -82,3 +82,22 @@ line-numbers = true
 
 [output.exerciser]
 output-directory = "comprehensive-rust-exercises"
+
+[output.i18n]
+default_language = "en"
+translate_all_languages = true
+optional = true
+
+[output.i18n.languages]
+"en" = "English"
+"es" = "Spanish (Español)"
+"ko" = "Korean (한국어)"
+"pt-BR" = "Brazilian Portuguese (Português do Brasil)"
+
+[output.tera-backend]
+after = "html"
+components = [
+    { name = "Macros", path = "src/components/macros.html" },
+    { name = "Included", path = "src/components/include_language_picker.html" },
+    { name = "LanguagePicker", path = "src/components/language_picker.html", dependencies = ["Included", "Macros"] },
```

Comment:
We need to declare dependencies here to include components or import macros. The syntax is the same for both

Reply:
Why can you not just include all files from a directory directly? Make them all available to Tera?

Reply:
It's an outdated comment it's not like that anymore